### ISSUE TITLE
Act as a node

### ISSUE THREAD

**Issue by duyvan1995** [Heavy-Vehicle-Networking-At-U-Tulsa/CAN-Logger-2#20]
Device does not act as a node even if listen only mode is set to false

**Comment by jeremy-daily** [comment]
Was this was fixed with the updated flexcan library?

**Comment by duyvan1995** [comment]
No, still did not see messages generated from a single CAN Logger, need to plug in a SSS2 to see those messages
